### PR TITLE
Кильдишев -- Лабораторная работа #1

### PR DESCRIPTION
**Pull request by mkildishev**
Hi, @SemyonBevzuk and @Nikita-Mitrokhin
I need a good review :)

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -2,6 +2,10 @@
 include "code/bevzuk-anton"
 project(":code/bevzuk-anton").name = "bevzuk-anton-Model"
 
+// Kildishev Maksim
+include "code/kildishev-maksim"
+project(":code/kildishev-maksim").name = "kildishev-maksim-Model"
```
Comment (SemyonBevzuk) [inline settings.gradle]: Not sure if everything is correct here.
The name of the project must match the theme of the laboratory work and be without personal information.

But I think it’s worth clarifying at a lecture.

File: code/kildishev-maksim/src/main/java/ru/unn/agile/figuresvolumecalculator/model/constants/MathConstants.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package ru.unn.agile.figuresvolumecalculator.model.constants;
+
+public final class MathConstants {
+
```
Comment (SemyonBevzuk) [inline code/kildishev-maksim/src/main/java/ru/unn/agile/figuresvolumecalculator/model/constants/MathConstants.java]: Better to make the MathConstants "package-private" class. It is needed only inside the package, but now you can use it from the outside.

File: code/kildishev-maksim/src/test/java/ru/unn/agile/figuresvolumecalculator/FiguresVolumeCalculatorTest.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+
+public class FiguresVolumeCalculatorTest {
+
+    private final double delta = 1e-2;
+
+    @Test
+    public void canCalculatePyramidVolume() {
+        assertEquals(3, FiguresVolumeCalculator.pyramidVolumeCalculate(3, 3), delta);
+    }
```
Comment (Nikita-Mitrokhin) [inline code/kildishev-maksim/src/test/java/ru/unn/agile/figuresvolumecalculator/FiguresVolumeCalculatorTest.java]: You use a lot of magic numbers, like here, fix it please

File: code/kildishev-maksim/src/main/java/ru/unn/agile/figuresvolumecalculator/model/constants/MathConstants.java (hunk 2 above)
Comment (mkildishev) [reply code/kildishev-maksim/src/main/java/ru/unn/agile/figuresvolumecalculator/model/constants/MathConstants.java]: This class placed in another package inside the main package

File: settings.gradle (hunk 1 above)
Comment (mkildishev) [reply settings.gradle]: Fixed!

File: code/kildishev-maksim/src/test/java/ru/unn/agile/figuresvolumecalculator/FiguresVolumeCalculatorTest.java (hunk 3 above)
Comment (mkildishev) [reply code/kildishev-maksim/src/test/java/ru/unn/agile/figuresvolumecalculator/FiguresVolumeCalculatorTest.java]: Fixed!

**Comment by Nikita-Mitrokhin** [comment]
:+1:

**Comment by SemyonBevzuk** [comment]
👍

**Comment by mkildishev** [comment]
@kirill-korniakov

**Comment by VitKonevskiy** [comment]
@mkildishev , Please, check the commit-history. Do you want to merge 60 commits? Confirm, that you rebase was successfull

**Review by VitKonevskiy — CHANGES_REQUESTED** [review]
I see at least 2 commits: e772a91 and df3b330 with the same code and title

**Comment by mkildishev** [comment]
@VitKonevskiy looks like in my first rebase attempt I accidentally made rebase from my branch to my branch. I'm sure that it is affect only commit history and revert is useless operation in this situation. Now rebase is made in the right way.

**Comment by VitKonevskiy** [comment]
@mkildishev I don't agree with you. After merge all these commits will be shown for everybody and can affect on clear understanding the branch-history. I insist, do rebase correctly, please.

**Comment by mkildishev** [comment]
@VitKonevskiy AFAIK, I can't just delete pushed commits. I can revert it, but it can make a more misunderstanding for anothers, IMHO. Also I can **squash** trash commits. Will it ok solution?

**Comment by vgvoleg** [comment]
@mkildishev no, you can. If there are incorrect commits, you can try interactive rebase and choose skip for them
Это весело, попробуй. А вообще я нормально отношусь к сквошу, если @VitKonevskiy убедился что разработка велась по тдд

**Comment by mkildishev** [comment]
@vgvoleg я бы сквошнул те, что остались после неудачного ребейза, изначальные не должны никуда деться. Если такой вариант устроит @VitKonevskiy, то сделаю так, иначе попробую ещё поиграться с ребейзами.
Спасибо, хотя бы буду знать в какую сторону копать.

**Comment by VitKonevskiy** [comment]
@mkildishev, по циклу tdd замечаний нет -- можешь сквошить, если тебе так проще, либо ребейз.

**Comment by mkildishev** [comment]
@VitKonevskiy посмотри ещё раз, плиз.

**Pull request merged**